hack/verify-shellcheck.sh

```diff
@@ -105,23 +108,23 @@ if ${HAVE_SHELLCHECK}; then
   shellcheck "${SHELLCHECK_OPTIONS[@]}" "${all_shell_scripts[@]}" || res=$?
 else
   echo "Using shellcheck ${SHELLCHECK_VERSION} docker image."
-  docker run \
+  "${DOCKER}" run \
     --rm -v "${KUBE_ROOT}:${KUBE_ROOT}" -w "${KUBE_ROOT}" \
     "${SHELLCHECK_IMAGE}" \
   shellcheck "${SHELLCHECK_OPTIONS[@]}" "${all_shell_scripts[@]}" || res=$?
 fi
 
 # print a message based on the result
 if [ $res -eq 0 ]; then
-  echo 'Congratulations! All shell files are passing lint (excluding those in hack/.shellcheck_failures).'
+  echo 'Congratulations! All shell files are passing lint :-)'
 else
   {
     echo
     echo 'Please review the above warnings. You can test via "./hack/verify-shellcheck.sh"'
     echo 'If the above warnings do not make sense, you can exempt this warning with a comment'
     echo ' (if your reviewer is okay with it).'
     echo 'In general please prefer to fix the error, we have already disabled specific lints'
-    echo ' that the project chooses to ignire.'
+    echo ' that the project chooses to ignore.'
```

Comment:
shellcheck, meet spellcheck :)

Reply:
we could do with a more complete spell checker.

Reply:
Please, no